toolset/benchmark/benchmarker.py

```diff
@@ -92,6 +92,9 @@ def __exit_test(self, success, prefix, file, message=None):
                 file=file,
                 color=Fore.RED if success else '')
         self.time_logger.log_test_end(log_prefix=prefix, file=file)
+        # Sleep for 60 seconds to ensure all host connects are closed
+        log("Clean up: Sleep 60 seconds...", prefix=prefix, file=file)
+        time.sleep(60)
```

Comment:
Even if it's the only one to run?

Reply:
Feel free to crtl-C if it's the only one run. Really, I just care about getting this on citrine and seeing if it solves the issues. Then I'll worry about QoL stuff.

Reply:
Fair enough.